dist/static/site.css

```diff
@@ -0,0 +1,366 @@
+  }
+  100% {
+    transform: translateX(30px);
+  }
+}
+
+#currentsong {
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+#currentplayer-descr {
+  font-family: "Righteous", monospace;
+  animation: 250ms pop-in ease, 400ms push-desc-out ease 250ms;
+  padding: 10px 0;
+  position: relative;
+  transform: translateX(30px);
+  text-shadow: 2px 2px 4px #000000;
+}
+
+@keyframes rotating-disc {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+@keyframes fade-in {
+  from,
+  40% {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+/* The disc animation */
+#currentplayer-disc {
+  position: absolute;
+  left: -30px;
+  animation: rotating-disc 3s linear infinite, fade-in 600ms ease;
+}
+
+.song img {
+  width: auto;
+  max-width: 120px;
+  height: 90px;
+  min-width: 90px;
+  opacity: 0.95;
+  float: left;
+}
+
+.song img:hover {
+  opacity: 1;
+}
+
+.songtitle {
+  height: 100%;
+  display: flex;
+  flex: 1;
+  align-self: center;
+  align-items: center;
+  margin: 10px;
+  max-width: 240px;
+}
+
+.songList > * {
+  margin: 5px 0;
+}
+
+/* Tab Menu */
+
+#tab-menu {
+  background: #111;
+  margin-top: 10px;
+  border-radius: 5px;
+}
+
+#tab-menu nav {
+  display: flex;
+  border-radius: 5px;
+}
+
+.menu-item {
+  display: inline-block;
+  font-weight: 600;
+  padding: 10px 20px;
+  cursor: pointer;
+  opacity: 0.6;
+  border-bottom: 4px solid transparent;
+  transition: 150ms ease;
+}
+
+.menu-item:hover,
+.menu-item.active {
+  opacity: 1;
+}
+
+.menu-item.active {
+  border-color: #a50bca;
+}
+
+#tab-queuesong,
+#tab-settings {
+  padding: 10px 20px;
+}
+
+#tab-queuesong label {
+  display: inline-block;
+  margin-top: 10px;
+}
+
+#input-songurl {
+  display: block;
+  padding: 8px 15px;
+  margin-top: 15px;
+  width: 100%;
+}
+
+#b-changename {
+  margin: 0.5em;
+}
+
+#volumecontrols {
+  display: flex;
+  align-items: center;
+}
+
+#volslider {
+  margin: 0.5em;
+}
+
+#noplaying,
+#disconnected {
+  display: none;
+  font-weight: bold;
+}
+
+#queuelist {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.queue {
+  flex: 1 0 auto;
+  margin: 5px;
+  max-width: 100%;
+}
+
+.errorbox {
+  color: #d93434;
+  display: none;
+  cursor: pointer;
+}
+
+.b-remove {
+  border: 0;
+  color: rgba(255, 255, 255, 0.6);
+  font-size: 16px;
+  background: transparent;
+  padding: 5px;
+  text-decoration: none;
+  margin: 0.5em;
+  align-self: center;
+  margin-left: auto;
+}
+
+#nowplaying .b-remove:before {
+  content: "Skip";
+}
+
+.b-remove:before {
+  content: "Remove";
+}
+
+.b-remove:hover {
+  background: #d93434;
+  color: #ffffff;
+  text-decoration: none;
+  padding: 5px 10px;
+}
+
+[hidden] {
+  display: none;
+}
+
+.fullscreen-bg {
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  overflow: hidden;
+  z-index: -1;
+}
+
+.fullscreen-bg__video {
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  min-width: 100%;
+  min-height: 100%;
+  transform: translate(-50%, -50%);
```

Comment:
Any particular reason why the video was translated `-50%` here and using `left`/`top: 50%`? Using the naive solution does seem to behave differently when resizing.

Reply:
Naive solution does not center on mobile, while this solution does.